httpx/_models.py

```diff
@@ -1473,7 +1473,7 @@ def __init__(self, response: Response):
         def info(self) -> email.message.Message:
             info = email.message.Message()
             for key, value in self.response.headers.multi_items():
-                #  Note that setting `info[key]` here is an "append" operation,
+                # Note that setting `info[key]` here is an "append" operation,
```

Comment:
there is one extra non-breakable space in this comment
https://github.com/encode/httpx/blob/07229b8dff61d3e9d819244d4835f547cae67b4d/httpx/_status_codes.py#L142